### ISSUE TITLE
Ubuntu, Small Mode, can't scroll pop-up list of Categories

### ISSUE THREAD

**Issue by q00u** [ValveSoftware/steam-for-linux#9850]
#### Your system information

* Steam client version (build number or date): 1689034492
* Distribution (e.g. Ubuntu): Ubuntu
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes
* Steam Logs: [steam-logs.tar.gz](https://github.com/ValveSoftware/steam-for-linux/files/12047821/steam-logs.tar.gz)
* GPU: Nvidia

#### Please describe your issue in as much detail as possible:
When trying to Categorize some games purchased in the recent sale, I discovered that the new Small Mode's pop-up list of Categories is not scrollable. Not with mousewheel, not with arrow keys. The scrollbar doesn't exist in Small Mode at all. I have a lot of Categories, so this makes it impossible to organize my library in Small Mode. In Large Mode, this works as expected.

![Screenshot from 2023-07-13 01-48-15](https://github.com/ValveSoftware/steam-for-linux/assets/892302/39230402-5a2b-4934-b48a-3f637206058c)

#### Steps for reproducing this issue:

0. Be in Small mode, on Linux
1. Select some games
2. Right click your selection
3. "Add selected to"
4. Try to scroll the list of Collections

tl;dr:  
"Add to categories" list can't scroll in Small mode. It _can_ in Large mode.
